acceptance/lib/helper.rb

```diff
@@ -1,4 +1,6 @@
 require 'beaker/dsl/install_utils'
+require 'puppetserver/acceptance/compat_utils'
+require 'puppetserver/acceptance/gem_utils'
```

Comment:
well, this bring up the original problem. The relative path to the helper is different depending upon how you're running the tests. (from pe-puppet-server-extensions or from puppet-server itself )